### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by nomadicoder**

### CODE REVIEW

**Comment by nomadicoder** [comment]
@bibliotechy Please proofread.

File: README.md [hunk 1]
```diff
@@ -1,48 +1,120 @@
-# README
+* `# README
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+Fortytude is an object oriented based website for the Temple University Libraries.
```
Comment (bibliotechy) [inline README.md]: Rails?

File: README.md [hunk 2]
```diff
@@ -1,48 +1,120 @@
...
 
-Things you may want to cover:
+## System Requirements
 
-* Ruby version
+- Ruby 2.5.1
+- Google OmniAuth2
```
Comment (bibliotechy) [inline README.md]: Is this a System Requirement? There is nothing special thay need to install, correct?

File: README.md [hunk 3]
```diff
@@ -1,48 +1,120 @@
...
 
-*Or* Create a account seed file 
+* Populate the database with fake data
 
-`mv db/account_seeds.rb.example db/account_seeds.rb`
+```
+bundle exec rake db:populate
+```
+
+* Seed initial user from the command line
```
Comment (bibliotechy) [inline README.md]: THis would be a goo place to be more explicit about the format the emails need to be in

File: README.md [hunk 4]
```diff
@@ -1,48 +1,120 @@
...
+To run continuous testing which will run the appropirate specs as you save
+source code, execute Guard
 
-* How to run the test suite
+```
+guard
+```
 
-* Services (job queues, cache servers, search engines, etc.)
+To perform mutation tests, which helps gage code coverage and test soundness on a class by class basis
```
Comment (bibliotechy) [inline README.md]: gage -> gauge

File: README.md [hunk 5]
```diff
@@ -1,48 +1,120 @@
...
-Install Ruby Gem dependencies
+* Set up environment variables:
 
-`bundle install`
+```
+export GOOGLE_OAUTH_CLIENT_ID="Google client ID goes here"
+export GOOGLE_OAUTH_SECRET="Google OAuth secret goes here"
+```
+Add these same lines to your `.bash_profile` or `.bashrc` file, depending on
+how you've setup your shee, depending on how you've setup your Bash shell:
```
Comment (bibliotechy) [inline README.md]: shee -> shell

Comment (bibliotechy) [reply README.md]: also, it rpeats the same clause twice.

File: README.md (hunk 2 above)
Comment (nomadicoder) [reply README.md]: When you do this on the Mac under RBEnv, it's hard to tell. I'll try provisioning it for a VM know for sure.

Comment (bibliotechy) [reply README.md]: I just mean Google Omniauth. Not ruby.

File: README.md (hunk 4 above)
Comment (nomadicoder) [reply README.md]: Check

File: README.md (hunk 1 above)
Comment (nomadicoder) [reply README.md]: ✔️

File: README.md (hunk 5 above)
Comment (nomadicoder) [reply README.md]: ✔️ ✔️

File: README.md (hunk 2 above)
Comment (nomadicoder) [reply README.md]: ✔️

File: README.md (hunk 3 above)
Comment (nomadicoder) [reply README.md]: ✔️

**Comment by nomadicoder** [comment]
@bibliotechy Done. Please review lines 45-46.

File: spec/examples.txt [hunk 6]
```diff
@@ -0,0 +1,128 @@
+example_id                                                     | status  | run_time        |
```
Comment (bibliotechy) [inline spec/examples.txt]: Why is this here?

Comment (bibliotechy) [reply spec/examples.txt]: @skng5 ping on this question

Comment (bibliotechy) [reply spec/examples.txt]: @nomadicoder ping on this question
